ivy/functional/frontends/tensorflow/layers.py

```diff
@@ -0,0 +1,6 @@
+#global
+import ivy
+import ivy.functional.frontends.tensorflow as tf_frontend

```

Comment:
you do not need this line as you are importing the farther folder of this file into this file, which will cause problems.